api/routes/editEntries.js

```diff
@@ -1,26 +1,26 @@
+//Edit an entry by Id
 const express =  require('express')
 const route = express.Router();
 const database = require('../models/post.model')
 
 
-route.patch('/:id', (req,res)=>{
-const {id} = req.params
-const {title, description} = req.body
-let match = false
-for(let i = 0; i < database.Posts.length; i++){
-if(id == database.Posts[i].id){
-match = true
-let updatedEntry = database.Posts[i].id
- updatedEntry = ({id:id, title:title, description:description, createdOn:Date()})
- database.Posts.splice(id , 1, updatedEntry)
-return res.json({status:"Success entry successfully updated!", data:updatedEntry})
-}
-}
-if(!match){
-  return res.json('Cannot find entry')
-}
-
-})
+ route.patch('/:id', (req,res)=>{
+ const {id} = req.params
+ const {title, description} = req.body
+ let match = false
+ for(let i = 0; i < database.Posts.length; i++){
+ if(id == database.Posts[i].id){
+ match = true
+ let updatedEntry = database.Posts[i].id
+  updatedEntry = ({id:id, title:title, description:description, createdOn:Date()})
+  database.Posts.splice(id , 1, updatedEntry)
```

Comment:
you don't need to Splice to update an entry. And you shouldn't change the createdOn time for an updated entry.